ckan/lib/helpers.py

```diff
@@ -110,6 +110,17 @@ def url(*args, **kw):
     return _add_i18n_to_url(my_url, locale=locale, **kw)
 
 
+def get_site_url_and_protocol():
```

Comment:
This returns (protocol, host). How about renaming this function `get_site_protocol_and_host()`

Also please add a docstring that shows that when for e.g. `site_url = http://example.com/prefix/` this helper returns `('http', 'example.com')`